### ISSUE TITLE
Can't download files

### ISSUE THREAD

**Issue by maximek3** [allenai/visual-reasoning-rationalization#1]
Hi there, I am getting `HTTP request sent, awaiting response... 403 Forbidden` for any of the file downloads (models, features and data). Any idea why that could be? Thanks!

**Comment by amarasovic** [comment]
Hey @maximek3, thank for pointing that out. Please can you try: `https://visual-reasoning-rationalization.s3-us-west-2.amazonaws.com/features.zip`; if that works I'll do the same for `models.zip` and `data.zip`. Thanks!

**Comment by maximek3** [comment]
Thanks for the response. That one seems to be working :)

**Comment by amarasovic** [comment]
Awesome, I updated the other links in the readme.

**Comment by maximek3** [comment]
Hi @amarasovic, I now downloaded all the files but can't seem to open then. I get the error `End-of-central-directory signature not found.  Either this file is not a zipfile, or it constitutes one disk of a multi-part archive.  In the latter case the central directory and zipfile comment will be found on the last disk(s) of this archive`. When I use the python `zipfile` library to open any of the files with a script, I get this error: `Exception has occurred: BadZipFile; File is not a zip file`. Any idea why that could be? Also, from the models link, I got one `models.zip` file that is 2.1GB and one `models.zip.1` file that is 122GB. I was wondering if this was supposed to be like that. Thanks a lot!
